lib/entry-list.ts

```diff
@@ -6,6 +6,29 @@ import {
   OnEntryListUpdatedHandlerReturn,
 } from './types'
 
+export default function createEntryList(channel: Channel): EntryListAPI {
+  let _handler: OnEntryListUpdatedHandler | null = null
+
+  channel.addHandler(
+    'entryListUpdated',
+    ({ msgId, props }: { msgId: string; props: OnEntryListUpdatedHandlerProps }) => {
+      return runHandler(_handler, props).then((result: OnEntryListUpdatedHandlerReturn) => {
+        return channel.send('entryListResult', { msgId, result })
+      })
+    }
```

Comment:
Nit: vs code proposes to convert this to an async function, maybe we can do that to get some newer-looking code